logging.tf

```diff
@@ -0,0 +1,85 @@
+# Base infra logging
+locals {
+  fluent_operator_namespace              = "fluent" # should we put it as a fluent_namespace variable ?
+  fluent_operator_helm_chart_version     = "v3.0.0" # variable ? and if "v3.0", then every terraform apply will update, even if there is no change
+  fluent_operator_cloudwatch_log_group   = "/${local.stack_name}-fluent-operator"
+  fluentbit_cloudwatch_log_stream_prefix = "fluentbit-"
+}
+
+# Fluent-operator (https://github.com/fluent/fluent-operator operator)
+resource "helm_release" "fluent_operator" {
+  chart      = "fluent-operator"
+  name       = "fluent-operator"
+  repository = "https://fluent.github.io/helm-charts"
+  version    = local.fluent_operator_helm_chart_version
+
+  create_namespace = true
+  namespace        = local.fluent_operator_namespace
+
+  values = [
+    <<-YAML
+    containerRuntime: containerd
+    fluentbit:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: node-role.kubernetes.io/edge
+                    operator: DoesNotExist
+                  - key: eks.amazonaws.com/compute-type
+                    operator: NotIn
+                    values:
+                      - fargate
+
+    YAML
+  ]
+}
+
+resource "aws_cloudwatch_log_group" "fluent_operator" {
```

Comment:
log group is related to fluent bit and not the operator

Reply:
not how I did it. it's the log stream which is for fluentbit, but the log group could be used by other app (fluentd for example?). Should we dedicate the log group to fluentbit ? What do you think @swibrow ?

Reply:
ok, finally dedicated the log group to fluentbit